prow/github/client.go

```diff
@@ -268,3 +268,25 @@ func (c *Client) CloseIssue(owner, repo string, number int) error {
 	}
 	return nil
 }
+
+// FindIssues uses the github search API to find issues which match a particular query.
+func (c *Client) FindIssues(query string) ([]Issue, error) {
```

Comment:
Add a simple test for this function.

We should consider accepting a `map[string][]string` (or `net/url.Values`) and using `net/url` to make the actual URL, but that can happen later.